gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,5 @@
-# checksum was taken from https://gradle.org/release-checksums
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=f6b8596b10cce501591e92f229816aa4046424f3b24d771751b06779d58c8ec4
```

Comment:
why are we removing this? I think it's better to leave it in and update the checksum